lib/src/utils/phone_number.dart

```diff
@@ -108,7 +108,7 @@ class PhoneNumber extends Equatable {
     if (prefix.isNotEmpty) {
       prefix = prefix.startsWith('+') ? prefix : '+$prefix';
       var country = Countries.countryList
-          .firstWhereOrNull((country) => country['dial_code'] == prefix);
+          .firstWhereOrNull((country) => prefix.contains(country['dial_code']);
```

Comment:
missing one more bracket at the end

Reply:
Hi!, no need more brackets, what are you talking about?

Reply:
at the end of 111

Reply:
After code'] is the )
```
.firstWhereOrNull((country) => prefix.contains(country['dial_code']);
```

Reply:
Oh you are right!